src/waveform/guitick.cpp

```diff
@@ -10,7 +10,7 @@ GuiTick::GuiTick(QObject* pParent)
     m_cpuTimer.start();
 }
 
-// this is called from the VSyncThread
+// this is called from WaveformWidgetFactory::render
```

Comment:
It would help to add a bit more to the comment: "This is called from WaveformWidgetFactory::render in the main thread"